qiskit/aqua/aqua_globals.py

```diff
@@ -31,6 +31,8 @@ class QiskitAquaGlobals:
 
     CPU_COUNT = local_hardware_info()['cpus']
 
+    LIBRARY_MSG = "The '{libname}' library is required to use '{name}'. {extra}"
```

Comment:
Would `MISSING_LIB_MSG` or so be a more precise name?

Reply:
Removed this.